### DISCUSSION TITLE
Allow Progress padding to be customizable

### DISCUSSION THREAD

**Discussion by sandrofigo** [spectreconsole/spectre.console#1450] [Ideas]
As already mentioned by this issue from another user https://github.com/spectreconsole/spectre.console/issues/1346, it would be nice to have a way to control the automatically inserted paddings above and below the rendered progress content.

I was thinking about adding an extension method in `ProgressExtensions` to modify the vertical padding that is then used in `DefaultProgressRenderer`: 

https://github.com/spectreconsole/spectre.console/blob/b21e07ea94543d33a6323523f2802944650d894d/src/Spectre.Console/Live/Progress/Renderers/DefaultProgressRenderer.cs#L113

Example:
```c#
AnsiConsole.Progress()
    .Padding(0)   // Adjust top and bottom padding
    .Start(ctx =>
    {
        // Omitted
    });
```

Similarly an extension method could be added for `Status` in `StatusExtensions`.

What do you think?

**Comment by f2bo** [comment]
Added additional information in the original issue https://github.com/spectreconsole/spectre.console/issues/1346.
